### ISSUE TITLE
Static rendering fails with Error: Not implemented: HTMLCanvasElement.prototype.getContext

### ISSUE THREAD

**Issue by nickjanssen** [frontarm/navi#48]
Hey there, navi rocks! ^^ 
My app is dependent on [react-native-vector-icons](https://github.com/oblador/react-native-vector-icons#web-with-webpack), which seems to fail navi-scripts's build process.

The full error:
```
Using create-react-app renderer...
Error: Not implemented: HTMLCanvasElement.prototype.getContext (without installing the canvas npm package)
    at module.exports (/Users/nickjanssen/Documents/docs/lk-web-app/node_modules/navi-scripts/node_modules/jsdom/lib/jsdom/browser/not-implemented.js:9:17)
    at HTMLCanvasElementImpl.getContext (/Users/nickjanssen/Documents/docs/lk-web-app/node_modules/navi-scripts/node_modules/jsdom/lib/jsdom/living/nodes/HTMLCanvasElement-impl.js:42:5)
    at HTMLCanvasElement.getContext (/Users/nickjanssen/Documents/docs/lk-web-app/node_modules/navi-scripts/node_modules/jsdom/lib/jsdom/living/generated/HTMLCanvasElement.js:41:47)
```

As the error indicated, I tried installing `canvas` in the root directory as well as in `node_modules/navi-scripts` but it doesn't have an effect. Any idea what I could be doing wrong?

Here's a repo with the reproducable error: https://github.com/nickjanssen/navi-test

**Comment by jamesknelson** [comment]
Thanks for submitting this!

It looks like the issue is a version mismatch -- jsdom 12.x expects canvas 1.x, and doesn't work with canvas 2.x.

I couldn't get canvas 1.x to install on my machine, but updating navi-scripts to use jsdom 13.x seems to fix the issue.

I'll need to do some more checking to make sure that updating to jsdom 13.x doesn't break anything else, but should be able to publish an update sometime during the week. In the meantime, if you manually add jsdom 13.x to your node_modules, and remove any `node_modules/navi-scripts/node_modules/jsdom` directory, then you should be able to get it to build as-is.

**Comment by jamesknelson** [comment]
I've just published 0.10.12, which bumps the jsdom version to 13.2.0, which should allow this to be solved by adding canvas 2.x to your package.json.
